Ambiguous-Squids/bangazon-api
# Created admin interface

Provide a general summary of your changes in the Title above

## Description
The description that you provide should be comprehensive enough to...

 Provide clarity to any potentially complex code.
 Explain reasons behind organizational or architectual decisions you made.
 Give context to what feature you were completing so that your teammate has a mental model before looking at the code.

## Related Issue
 Please link to the issue number here: 
 This project only accepts pull requests related to open issues 
 If suggesting a new feature or change, please discuss it in an issue first 
 If fixing a bug, there should be an issue describing it with steps to reproduce 

## Motivation and Context
 Why is this change required? What problem does it solve? 

## How Has This Been Tested?
 Please describe in detail how you tested your changes. 
 Include details of your testing environment, and the tests you ran to 
 see how your change affects other areas of the code, etc. 

## Steps to Test
You must provide clear steps for any teammate to test the code.

System configuration.
3rd party libraries that need to be installed.
Command line utilities to run.
If there is a UI component, give clear instructions for steps to perform in the UI, and what they should expect to see as the outcome of those steps.

## Types of changes
 What types of changes does your code introduce? Put an \`x\` in all the boxes that apply: 

## Checklist:
 Go over all the following points, and put an \`x\` in all the boxes that apply. 
 If you're unsure about any of these, don't hesitate to ask. We're here to help!
